src/lro-poller.ts

```diff
@@ -0,0 +1,78 @@
+import { Queue } from "./throttler/queue";
+import * as api from "./api";
+import * as FirebaseError from "./error";
+
+export interface LroPollerOptions {
+  pollerName?: string;
+  apiOrigin: string;
+  apiVersion: "v1" | "v1beta1";
+  operationResourceName: string;
+  backoff?: number;
+  masterTimeout?: number;
+}
+
+export interface LroResult {
+  error?: FirebaseError;
+  response?: any;
+}
+
+const DEFAULT_INITIAL_BACKOFF_DELAY_MILLIS = 250;
+const DEFAULT_MASTER_TIMEOUT_MILLIS = 30000;
+
+export class LroPoller {
```

Comment:
Can we just call this `OperationPoller`?

Reply:
I'm okay with this. @nbegley, how about you?

Reply:
SGTM

Reply:
Renamed `LroPoller` to `OperationPoller`